input/dispatch/dispatch.go

```diff
@@ -14,7 +14,7 @@ func Get(desire def.Input) input.Input {
 	case "dir":
 		input = dir.New(desire)
 	case "tar":
-		input = tar.New(desire)
+		input = tar2.New(desire)
```

Comment:
"exec-tar"?  No way of knowing if whatever may be on the $path is gnu or... what.